react/features/device-selection/components/AudioDevicesSelection.web.tsx

```diff
@@ -248,16 +277,86 @@ class AudioDevicesSelection extends AbstractDialogTab<IProps, IState> {
 
         const classes = withStyles.getClasses(this.props);
 
+        const isAudioSettingsEnabled = Boolean(audioSettings?.autoGainControl || audioSettings?.channelCount === 2 || audioSettings?.echoCancellation || audioSettings?.noiseSuppression);
+
         return (
             <div className = { classes.container }>
                 {!iAmVisitor && <div
                     aria-live = 'polite'
                     className = { classes.inputContainer }>
                     {this._renderSelector(audioInput)}
                 </div>}
+
                 {!hideAudioInputPreview && hasAudioPermission && !iAmVisitor
-                        && <AudioInputPreview
-                            track = { this.state.previewAudioTrack } />}
+                                    && <AudioInputPreview
+                                        track = { this.props.previewAudioTrack } />}
+
+
+                {audioSettings && !hideNoiseSuppression && !iAmVisitor && (
```

Comment:
Please show these checkboxes conditionally based on `enableAdvancedAudioSettings`  settings in config.js. You can add a helper function in https://github.com/jitsi/jitsi-meet/blob/3d83847e4be8cf3ff1ac27ae879288d4226cc947/react/features/base/config/functions.any.js for checking the flag.